apps/meteor/ee/server/lib/deviceManagement/session.ts

```diff
@@ -84,13 +84,15 @@ export const listenSessionLogin = async (): Promise<void> => {
 			}
 
 			try {
-				Mailer.send({
-					to: `${name} <${email}>`,
-					from: Accounts.emailTemplates.from,
-					subject: settings.get('Device_Management_Email_Subject'),
-					html: mailTemplates,
-					data: mailData,
-				});
+				const isLoginEmailEnabled = settings.get('Enable_Login_Emails');
```

Comment:
A setting will disable this feature for every user at once. I think this should be a user preference instead of an admin setting. The admin can change the default value of the preference if they want but the user will still be in control of his own security.

Reply:
Done! Please take a look.